src/rhash/hash.c

```diff
@@ -4015,6 +4031,10 @@ void rc_hash_initialize_iterator(struct rc_hash_iterator* iterator, const char*
         {
           iterator->consoles[0] = RC_CONSOLE_APPLE_II;
         }
+        else if (rc_path_compare_extension(ext, "wav"))
```

Comment:
I'm wary that `.wav` files wouldn't be exactly identical for multiple users.

Reply:
It's apparently just used to be interpreted as a tape. I'm not sure how common dumps are actually using a .wav file here.

Reply:
Presumably different recordings of old tapes would result in different .wav files, so this only makes sense if there's a "canonical" set of .wav recordings .. otherwise, maybe best not to support .wav, since there are other formats that might make more sense?